charts/appdaemon/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for Appdaemon.
+
+image:
+  repository: acockburn/appdaemon
+  pullPolicy: IfNotPresent
+  tag: 4.0.5
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5050
+  # # Boilerplate config for code-server functionality, be sure to make changes to suit your environment
```

Comment:
Boilerplate config for vscode-server?

Reply:
Is it called code-server? I'm not sure what is correct here. ;)

Reply:
It is called code-server :)

https://github.com/cdr/code-server